libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFDirectoryManager.h

```diff
@@ -94,15 +94,15 @@ FOUNDATION_EXTERN NSString * const kDefaultCommunityName NS_SWIFT_NAME(SFDirecto
  @param components The additional path components to be added at the end of the directory (eg ['mybundle', 'common'])
  @return The path to the directory
  */
-- (NSString*)directoryOfCurrentUserForType:(NSSearchPathDirectory)type components:(NSArray*)components;
+- (NSString*)directoryOfCurrentUserForType:(NSSearchPathDirectory)type components:(nullable NSArray*)components;
```

Comment:
I think this one and the one below have nullable return strings too

Reply:
You are right! Fixed.